fiftyone/utils/databricks.py

```diff
@@ -0,0 +1,18 @@
+"""
+Databricks utilities.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+import fiftyone as fo
```

Comment:
This might cause a circular dep...we only really need the __version__.

Reply:
This indeed looks like a circular dep to me